### ISSUE TITLE
Problems with arp2db command

### ISSUE THREAD

**Issue by astramax** [cea-sec/ivre#459]
### IVRE Version

ivre installed by pip

```
# ivre
IVRE - Network recon framework
Copyright 2011 - 2017 Pierre LALET <pierre.lalet@cea.fr>
Version 0.9.6

Python 2.7.14 (default, Sep 17 2017, 18:50:44) 
[GCC 7.2.0]

Dependencies:
    Python module Crypto: 2.6.1
    Python module pymongo: 3.5.1
    Python module py2neo: 3.1.2
    Python module sqlalchemy: 1.1.11
    Python module psycopg2: 2.7.3 (dt dec mx pq3 ext lo64)
    Python module krbV: missing
    Python module PIL: 4.3.0
    Python module MySQLdb: 1.3.10
    Python module dbus: 1.2.4
    Python module matplotlib: 2.0.0
```

### Summary

Problems with arp2db command

### Expected behavior

I want to analyze flows from cap. file with arp-trafic

### Actual behavior

i am try to execute
```
ivre arp2db ./prg/arp_4h.pcap 
```
but get
```
reading from file ./prg/arp_4h.pcap, link-type EN10MB (Ethernet)
WARNING:scapy.runtime:PcapReader: unknown LL type [1]/[0x1]. Using Raw packets
Traceback (most recent call last):
  File "/usr/local/bin/ivre", line 78, in <module>
    main()
  File "/usr/local/bin/ivre", line 51, in main
    tools.get_command(possible_commands[0])()
  File "/usr/local/lib/python2.7/dist-packages/ivre/tools/arp2db.py", line 60, in main
    for pkt in reader(fname):
  File "/usr/lib/python2.7/dist-packages/scapy/utils.py", line 646, in next
    pkt = self.read_packet()
  File "/usr/lib/python2.7/dist-packages/scapy/utils.py", line 730, in read_packet
    p = conf.raw_layer(s)
TypeError: 'NoneType' object is not callable
```
may be it scapy's bug? but i add 2 lines to arp2db.py:
```
from scapy.layers import l2
from scapy.layers import inet
```
and fixed this bug.
 
Then i try to execute "ivre arp2db ./prg/arp_4h.pcap" one more time, but get
```
# ivre arp2db ./prg/arp_4h.pcap 
WARNING:scapy.runtime:No route found for IPv6 destination :: (no default route?)
reading from file ./prg/arp_4h.pcap, link-type EN10MB (Ethernet)
Traceback (most recent call last):
  File "/usr/local/bin/ivre", line 78, in <module>
    main()
  File "/usr/local/bin/ivre", line 51, in main
    tools.get_command(possible_commands[0])()
  File "/usr/local/lib/python2.7/dist-packages/ivre/tools/arp2db.py", line 67, in main
    db.flow.bulk_add_flow(
AttributeError: 'Neo4jDBFlow' object has no attribute 'bulk_add_flow'
```
and class [Neo4jDBFlow](https://github.com/cea-sec/ivre/blob/master/ivre/db/neo4j.py) really haven't method bulk_add_flow

**Comment by p-l-** [comment]
I don't think `arp2db` is actually used...

Can you replace
```Python
db.flow.bulk_add_flow(
                        bulk, rec, "ARP", {},
[...]
```
with
```Python
db.flow.add_flow(
                        rec, "ARP", {},
[...]
```
and report here the result?

**Comment by astramax** [comment]
Thanks! I've done this and have  no errors:

```
root@kali:~# ivre arp2db -v ./prg/arp_4h.pcap 
WARNING:scapy.runtime:No route found for IPv6 destination :: (no default route?)
reading from file ./prg/arp_4h.pcap, link-type EN10MB (Ethernet)
root@kali:~# 
```
but no one flow found in db (pcap contains over 900 arp-packets):
```
root@kali:~# ivre flowcli -c
0 clients
0 servers
0 flows
root@kali:~# ^C
```

**Comment by astramax** [comment]
I try another way: proceed pcap with argus and than add flows by flow2db.
But get:
```
root@kali:~# ivre flow2db -v ./prg/flows_arp.argus 
Traceback (most recent call last):
  File "/usr/local/bin/ivre", line 78, in <module>
    main()
  File "/usr/local/bin/ivre", line 51, in main
    tools.get_command(possible_commands[0])()
  File "/usr/local/lib/python2.7/dist-packages/ivre/tools/flow2db.py", line 108, in main
    for rec in fdesc:
TypeError: iter() returned non-iterator of type 'Argus'
root@kali:~# 
```
Argus's flows-file is [flows_argus_arp.txt](https://github.com/cea-sec/ivre/files/1645529/flows_argus_arp.txt)

**Comment by p-l-** [comment]
If you just want the flow data and have a PCAP file, the easiest way is to process the PCAP with `bro` using the `flow` module, then use `ivre bro2db` with all the generated log files.

**Comment by p-l-** [comment]
Also, if you want to try with Argus, you should use the current development version.

The `flow2db` is currently broken with Argus but only with Python 3.

**Comment by p-l-** [comment]
Can you also share the PCAP file so that I can look why `arp2db` does not work?

**Comment by p-l-** [comment]
The PR #466 should fix your issues with `arp2db` and should fix `flow2db` with Argus files on Python 3.

**Comment by p-l-** [comment]
@astramax this has been closed because #466 has been merged, but please comment here if it does not (or if it does) fixes thinks for you.

**Comment by astramax** [comment]
bug fixed. thanks!
```
# ivre
IVRE - Network recon framework
Copyright 2011 - 2017 Pierre LALET <pierre.lalet@cea.fr>
Version 6b68a54

Python 2.7.14+ (default, Dec  5 2017, 15:17:02) 
[GCC 7.2.0]

Linux kali 4.13.0-kali1-amd64 #1 SMP Debian 4.13.10-1kali2 (2017-11-08) x86_64

Dependencies:
    Python module Crypto: 2.6.1
    Python module pymongo: 3.5.1
    Python module py2neo: 3.1.2
    Python module sqlalchemy: 1.1.11
    Python module psycopg2: 2.7.3 (dt dec mx pq3 ext lo64)
    Python module krbV: missing
    Python module PIL: 4.3.0
    Python module MySQLdb: 1.3.10
    Python module dbus: 1.2.4
    Python module matplotlib: 2.0.0
```
result of # ivre arp2db ./prg/arp_4h.pcap  is
```
# ivre flowcli -c
18 clients
750 servers
757 flows
```
arp-trafic attached
[arp_4h.zip](https://github.com/cea-sec/ivre/files/1649685/arp_4h.zip)
